generate_wordstr_box.py

```diff
@@ -16,26 +17,19 @@
 # Image file
 arg_parser.add_argument('-i', '--image', nargs='?', metavar='IMAGE', help='Image file', required=True)
 
+# Reading direction
+arg_parser.add_argument('-r', '--rtl', action='store_true', default=False, help='RTL input')
+
 args = arg_parser.parse_args()
 
 #
 # main
 #
 
-def trim(im):
```

Comment:
So you no longer suggest to use trimming. Why?

Reply:
No need to add additional overhead of trimming specially when dealing with thousands of images. Assumption is that images and ground-truth are pre-processed to meet training needs.

Reply:
Done via https://github.com/tesseract-ocr/tesstrain/pull/160